src/ui/window.c

```diff
@@ -1214,7 +1214,7 @@ win_print_outgoing(ProfWin* window, const char* show_char, const char* const id,
         _win_correct(window, message, id, replace_id, myjid);
     } else {
         //TODO my jid
```

Comment:
What is this about?

Reply:
I'm not entirely sure anymore. At one point I rewrote window functions to have more infos inside the function like the whole jid and not just a part of it given as *char. I'm not sure whether this TODO is up to date or just forgot to remove it. In https://github.com/profanity-im/profanity/commit/f7fe87dd4e7e14112fbcb0ae8794d90ffee64f2f it was added and in outgoing_muc() too but there it got removed later. So I guess I just forgot to remove it it.

Reply:
I did a new commit which explains and removes this TODO :)